lib/components/RadioButton/RadioButton.tsx

```diff
@@ -15,7 +15,7 @@ export const RadioButton = ({ label, value, className }: RadioButtonProps) => {
       {({ checked }) => (
         <div className="ds-flex-start ds-flex ds-space-x-4">
           {checked ? <CheckedIcon /> : <UncheckedIcon />}
-          <span className="ds-flex ds-items-center ds-text-button-md ds-text-black hover:ds-text-accent hover:ds-underline ds-hyphens-auto">
+          <span className="ds-flex ds-items-center ds-text-heading-4 ds-text-black hover:ds-text-accent hover:ds-underline ds-hyphens-auto">
```

Comment:
I'd really like to stick with `sm`, `md` , `lg` etc. The difference between `ds-text-button-md` and `ds-text-heading-4` is the `letter-spacing`. It's 0.16px in `text-button-md`, which is almost nothing. I even made a comparison (top row has the letter-spacing: 0.16px) 🙃

![image](https://github.com/user-attachments/assets/fc1bce81-0694-41ae-9776-14e371870af8)

Reply:
It is a small difference for sure, but the change was made so that the code/implementation matches with the design where it is explicitly told to use this heading-4

From Figma:
```
/* jod - headline 4 */
font-family: Poppins;
font-size: 16px;
font-style: normal;
font-weight: 600;
line-height: 24px; /* 150% */
```